tests/h/services/list_organizations_test.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.services.list_organizations import ListOrganizationsService
+from h.services.list_organizations import list_organizations_factory
+from h.services.organization import organization_factory
+
+
+class TestListOrganizations(object):
+
+    def test_returns_organizations_if_no_user(self, svc, organizations, alternate_organizations):
+        results = svc.organizations()
+
+        assert '__default__' in [org.pubid for org in results]
+        assert not (set(organizations) - set(results))
+
+    def test_returns_organizations_if_user(self,
```

Comment:
This test throws a different authority at the service—which is something that's good to test—but its naming related to `user` is confusing.